### PR TITLE
San 4757 container inspect

### PR DESCRIPTION
**Pull request by podviaznikov**
- add new task `container.state.poll`
- which would emit `container.state.polled` event on success

The plan further:
api would handle `docker.container.polled` event.
khronos network canary would trigger something like `instance.container.health-check.failed` which DL would pickup and create `docker.container.poll` command.

### CODE REVIEW

File: lib/workers/docker.container.poll.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+const rabbitmq = require('../rabbitmq')
+
+module.exports.jobSchema = Joi.object({
+  host: Joi.string().required(),
+  id: Joi.string().required(),
+  tid: Joi.string()
+}).unknown().required().label('DockerContainerPoll job')
+
+module.exports.task = (job) => {
+  const log = logger({ job: job })
```
Comment (anandkumarpatel) [inline lib/workers/docker.container.poll.js]: no need to log job anymore since we have transactions :)

File: lib/docker-utils.js [hunk 2]
```diff
@@ -34,3 +35,51 @@ module.exports.testEvent = function (dockerClient) {
...
+ */
+module.exports._handleInspectError = (host, err, log) => {
+  if (err.statusCode === 404) {
+    // container is not there anymore. Exit
+    const fatalErr = new WorkerStopError(
+      err.message,
+      { originalError: err }, { level: 'info' }
+    )
+    log.trace({ err: err }, '_handleInspectError - container not found')
+    throw fatalErr
```
Comment (anandkumarpatel) [inline lib/docker-utils.js:61]: We dont have to do this right now, but would we want to emit `container.lost` event?

Comment (podviaznikov) [inline lib/docker-utils.js:61]: yeah, that sounds like a good idea. We definitely should handle that

**Comment by anandkumarpatel** [comment]
code looks good!

**Comment by podviaznikov** [comment]
tested on gamma manually creating job with `tid` `some-tid`. Everything worked.

**Pull request merged**